chat-commands.js

```diff
@@ -759,7 +759,7 @@ const commands = {
 		}
 
 		if (room.isPersonal) {
-			if (!this.can('editroom', null, room)) return;
+			if (!this.can('declare', null, room)) return;
```

Comment:
lol @bumbadadabum I did say I was pretty sure `editroom` was used for something.

Anyway, `declare` is a really non-sequitur permission here, I'd probably suggest adding a new permission or something.

Reply:
I don't really like adding permissions since there's no way to update other people's group config, but if you think it's ok then ill do that.

Reply:
Can you think of a Host permission that fits better?

Reply:
I think it gets `tournamentsmanagement`, I would put it on that.